lib/src/ssl_funcs/Md5.cc

```diff
@@ -253,10 +252,7 @@ std::string Md5Encode::encode(const std::string &src_info)
         data_BIT_OF_GROUP += i * (BIT_OF_GROUP / BIT_OF_BYTE);
         RotationCalculate(data_BIT_OF_GROUP, param);
     }
-    if (NULL != out_data_ptr)
-    {
-        delete[] out_data_ptr, out_data_ptr = NULL;
-    }
+    delete[] out_data_ptr, out_data_ptr = NULL;
```

Comment:
IMO, explicitly using `delete` is not modern. `FillData` should return `std::vector<char>`.